tests/next_tests/unit_tests/conftest.py

```diff
@@ -23,12 +24,22 @@
 from gt4py.next.iterator import ir as itir, pretty_parser, pretty_printer, runtime, transforms
 from gt4py.next.program_processors import processor_interface as ppi
 from gt4py.next.program_processors.formatters import gtfn, lisp, type_check
-from gt4py.next.program_processors.runners import (
-    dace_iterator,
-    double_roundtrip,
-    gtfn_cpu,
-    roundtrip,
-)
+from gt4py.next.program_processors.runners import double_roundtrip, gtfn_cpu, roundtrip
+
+
+try:
+    import dace
```

Comment:
Is this explicit dace import really needed? The `import dace` inside `dace_iterator` should produce the same exception if it is not installed, right?

Reply:
This error disappears after deleting the Python venv and creating a new one.